### PR TITLE
feat(make): error uniqueness check

### PR DESCRIPTION
**Pull request by georgettica**
this will make the CONTRIBUTING.md cleaner, and is added to the 'make ci-check'

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,6 +37,17 @@ cadctl-install-local-force:
...
 
+
+# will hold all of the checks that will run on the repo. this can be extracted to a script if need be
+.PHONY: checks
+checks: check-duplicate-error-messages
+
+# check-duplicate-error-messages will conform with 
+.PHONY: check-duplicate-error-messages
+check-duplicate-error-messages:
+	@(test $$(grep -Ir errors.New . | grep -v -e ./CONTRIBUTING.md | wc -l) -eq 0) || (echo "please use fmt.Errorf and not errors.New" >&2 && exit 1)
```
Comment (shibumi) [inline Makefile:48]: Learning question: Why do we prefer fmt.Errorf over errors.New()?
Do we just want to get rid of injecting another dependency in our package that already uses fmt?

Comment (georgettica) [reply Makefile:48]: not quite, it's just that fmt.Errorf allows for errors.New AND wrapping errors `fmt.Errorf("oh no! %w", err)`

**Comment by shibumi** [comment]
@georgettica  can we merge this PR or is anything particularly blocking this PR?

**Comment by georgettica** [comment]
There isn't, it just wasn't in the planned sprint :(

**Comment by georgettica** [comment]
/lgtm
/approve
I guess I can't as I am the only approver aside of @rafael-azevedo and I cannot self approve my stuff

**Comment by georgettica** [comment]
@shibumi see the second commit, it shows we had some duplicate error msgs :)

**Pull request merged**